monkey_island/cc/resources/telemetry.py

```diff
@@ -124,6 +125,8 @@ def process_exploit_telemetry(telemetry_json):
         if new_exploit['result']:
             EdgeService.set_edge_exploited(edge)
 
+        Telemetry.encrypt_exploit_creds(telemetry_json)
```

Comment:
At this point, what happened to the plain text creds in the `telemetry_json` which were written to the mongodb under `new_exploit`?

Reply:
If you're asking what happened up to this point, nothing happened. They were sent plain text from the monkey (over HTTPS).

Reply:
Meant the line where mongo.db.edge.update
With something that's a deep clone of telemetry_json['data']
which is what's being encrypted later.

We're encrypting the data when we add the creds but it seems the data is also saved without encryption under edges.